.ci/run_tests.sh

```diff
@@ -52,7 +52,8 @@ if [ "$TEST_SUITE" == "dbless" ]; then
     eval "$TEST_CMD" spec/02-integration/02-cmd \
                      spec/02-integration/05-proxy \
                      spec/02-integration/04-admin_api/02-kong_routes_spec.lua \
-                     spec/02-integration/04-admin_api/15-off_spec.lua
+                     spec/02-integration/04-admin_api/15-off_spec.lua \
+                     spec/02-integration/08-status_api/03-readiness_endpoint_spec.lua
```

Comment:
This seems wrong.
This test must be run on all modes of Kong - traditional, hybrid and DB-less.

Reply:
> This seems wrong. This test must be run on all modes of Kong - traditional, hybrid and DB-less.

It was already like this. See this
https://github.com/Kong/kong/actions/runs/4663265543/jobs/8254440772?pr=10610